builds/misc/templates/base-image-publish/base-image-overview.yaml

```diff
@@ -0,0 +1,26 @@
+steps:
+- pwsh: |
+    # Note: With in the same job, you can access $(info), but it would interpret $(info) as string similar to `echo "$info"` which is undesirable for our case, we want an Object.
+    # We worked around this by using the serialization & deserailzation
+    $info = '$(serialize_info)' | ConvertFrom-Json
+
+    # Fetch tags & version from each module Dockerfiles {../$(arch)/base/Dockerfile, ../$(arch)/Dockerfile}
+    foreach ($each in $info)
+    {
+      $pathPrefix = "$(Build.SourcesDirectory)/$($each.Item1.ToString())/docker/$(os)/$(arch)"
+      $unberlyingFilePath = $pathPrefix + '/base/Dockerfile' | Resolve-path
+      $underlyingTag = $($(Get-Content $unberlyingFilePath | Select-String "ARG base_tag=") -split "=")[1]
+      $imageFilePath = $pathPrefix + '/Dockerfile' | Resolve-path
+      Set-Variable -Name "$($each.Item3.ToString())" -Value "$($($(Get-Content $imageFilePath | Select-String "ARG base_tag=") -split "=")[1])"
+      Write-Output "$($each.Item2.ToString()):$(Get-Variable -Name "$($each.Item3.ToString())" -ValueOnly) <= $underlyingTag"
+      Write-Output "##vso[task.setvariable variable=$($each.Item3.ToString())]$(Get-Variable -Name "$($each.Item3.ToString())" -ValueOnly)"
```

Comment:
Why do we set the var to itself? Or am I missing something?

Reply:
The wonder of VSTS variable. This is actually setting the variable to pipeline's JOB scope. Which is different that task env var. As it turns out, we don't need this var, so I'm deleting it.